### PR TITLE
Animate addition of a new recording

### PR DESCRIPTION
**Pull request by microbit-grace**
Animation for when the graph and finger print is added is not ideal, but the design for the fingerprint part is not finalised

### CODE REVIEW

**Comment by microbit-grace** [comment]
Putting back into WIP because I think I'll add the extra done recording state into this PR

File: src/components/DataRecordingGridItem/styles.module.css [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  animation: flash 2s;
+}
+@-webkit-keyframes flash {
+  0%,
+  10% {
+    background-color: #4040ff44;
+  }
+  100% {
+  }
+}
```
Comment (microbit-matt-hillsdon) [inline src/components/DataRecordingGridItem/styles.module.css:12]: The prefixing should happen my magic where necessary.

```suggestion
.flash-animation {
  animation: flash 2s;
}
```

File: src/components/DataSampleGridView.tsx [hunk 2]
```diff
@@ -51,27 +58,39 @@ const DataSamplesGridView = () => {
...
+  const [newRecordingId, setNewRecordingId] = useState<number | undefined>(
+    undefined
+  );
+  const handleRecordingComplete = useCallback((recordingId: number) => {
+    setNewRecordingId(recordingId);
+  }, []);
+
+  const handleOpenRecordingDialog = useCallback(() => {
+    recordingDialogDisclosure.onOpen();
+  }, [recordingDialogDisclosure]);
```
Comment (microbit-matt-hillsdon) [inline src/components/DataSampleGridView.tsx:79]: Both of these can be replaced by the functions they call.

**Review by microbit-matt-hillsdon — CHANGES_REQUESTED** [review]
Code feedback but this works well, thanks.

File: src/components/DataRecordingGridItem/styles.module.css (hunk 1 above)
Comment (microbit-grace) [reply src/components/DataRecordingGridItem/styles.module.css]: 👍  [82643d2](https://github.com/microbit-foundation/ml-trainer/pull/420/commits/82643d2f528e8b9308bc02786f02e989fa1d14e4)

File: src/components/DataSampleGridView.tsx (hunk 2 above)
Comment (microbit-grace) [reply src/components/DataSampleGridView.tsx]: [02fb37b](https://github.com/microbit-foundation/ml-trainer/pull/420/commits/02fb37bbc1160ba277c825902b237131982605a1)

**Review by microbit-matt-hillsdon — APPROVED** [review]
LGTM, again I can imagine we'll get more feedback or better ideas but let's see.

**Pull request merged**